app/src/main/java/com/safetyheads/akademiaandroida/splashscreen/SplashScreenUseCaseImpl.kt

```diff
@@ -6,6 +6,6 @@ import kotlinx.coroutines.delay
 class SplashScreenUseCaseImpl: SplashScreenUseCase {
```

Comment:
So for UseCase we don't need `interface` :) also you can move it to domain module.